army schema work/1 table.sql

```diff
@@ -0,0 +1,42 @@
+use ArmyDB
+go
+
+drop table if exists dbo.Soldier
+go 
+
+create table dbo.Soldier
+(
```

Comment:
-- LL missing a primary key here

Reply:
-- LL the primary key is misspelled, please correct so that it doesn't cause you problems in the future